MarvinLee_Portfolio_FEND/css/main.css

```diff
@@ -102,29 +100,21 @@ figcaption {
   font-family: "Courier New", Courier, monospace;
 }
 
-.featured_images_left {
-  order: -1;
-  width: 33.3333%;
+.featured_image {
+  width: 32%;
   margin: 0 0 0 0;
 }
 
-.featured_images_center {
-  order: 0;
-  width: 33.3333%;
-  margin: 0 auto;
+.featured_image img {
+  width: 100%;
 }
 
-.featured_images_right {
-  order: 1;
-  width: 33.3333%;
-  margin: auto 0 auto 0;
-}
 

```

Comment:
All 3 above have the same style. Ordering them is unnecessary. I have set them to 32% and flexbox will fill the gab